app/api/services/suppliers.py

```diff
@@ -38,3 +38,24 @@ def get_suppliers_by_name_keyword(self, keyword):
                     raiseload('*'))
                 .limit(20)
                 .all())
+
+    def get_metrics(self):
+        supplier_count = (
+            db
+            .session
+            .query(
+                Supplier.id
+            )
+            .outerjoin(SupplierFramework)
+            .outerjoin(Framework)
+            .filter(
+                Supplier.abn != Supplier.DUMMY_ABN,
+                Supplier.status != 'deleted',
+                or_(Framework.slug == 'digital-marketplace', ~Supplier.frameworks.any())
+            )
+            .count()
```

Comment:
FWIW it looks like using the `func.count()` method might be faster: https://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.count